src/platform/user/profile/vet360/util/local-vet360.js

```diff
@@ -227,6 +227,56 @@ export default {
       1000,
     );
   },
+  addressValidationSuccessTwoConfirmedSuggestions() {
```

Comment:
That function name, tho.

Reply:
why does everyone have a problem with long function names. fwiw this code is only run when testing locally.